pkg/prometheus/testdata/KeepDroppedTargets.golden

```diff
@@ -4,6 +4,7 @@ global:
   external_labels:
     prometheus: default/test
     prometheus_replica: $(POD_NAME)
+  keep_dropped_targets: 1000
```

Comment:
Was this change intentional? Do we want to make the same changes to `keep_dropped_targets`?

Reply:
From what me and @simonpasquier discussed, this should be a consistent behaviour through all the limits.